benchexec/tools/klee.py

```diff
@@ -52,6 +59,15 @@ def version(self, executable):
         line = line.split('(')[0]
         return line.strip()
 
+    def cmdline(self, executable, options, tasks, propertyfile=None, rlimits={}):
+        if "memlimit" in rlimits:
+            options += ["--max-memory="+str(rlimits["memlimit"])]
+        if "walltimelimit" in rlimits:
+            options += ["--max-walltime="+str(rlimits["walltimelimit"])]
+        if "timelimit" in rlimits:
```

Comment:
Probably you want to check for `softtimelimit` here. The soft time limit defines how long a run should take at most (results produced later are not counted), whereas `timelimit` is the limit after which the tool is forcefully killed.